src/test/java/net/fabricmc/stitch/tinyv2tests/TestTinyV2ReadAndWrite.java

```diff
@@ -0,0 +1,39 @@
+package net.fabricmc.stitch.tinyv2tests;
+
+import net.fabricmc.stitch.commands.tinyv2.TinyFile;
+import net.fabricmc.stitch.commands.tinyv2.TinyV2Reader;
+import net.fabricmc.stitch.commands.tinyv2.TinyV2Writer;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class TestTinyV2ReadAndWrite {
+    private static final String DIR = "src/test/resources/sorted/";
```

Comment:
Uh, this is not right! Use `TestTinyV2ReadAndWrite.class.getResource`

Reply:
I swear to god I get a ClassNotFoundException by calling that

Reply:
tell me, why the fuck does this return null?
`TestTinyV2ReadAndWrite.class.getResource("sorted")`

Reply:
and then if I do `TestTinyV2ReadAndWrite.class.getClassLoader().getResource("sorted").getFile()`
I get this path `/C:/Users/natan/Desktop/stitch/build/resources/test/`
which is unacceptable by java because of the leading `/`

Reply:
same thing as with .getPath()

Reply:
here's one that actually works
```
    private static final String DIR = new File(TestTinyV2ReadAndWrite.class.getClassLoader().getResource("sorted").getPath()).getAbsolutePath() + "/";
```